app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -377,6 +392,14 @@
                 android:layout_marginTop="@dimen/spacing_tiny"
                 android:text="@string/take_more_pictures" />
 
+            <Button
+                android:id="@+id/add_nutrition_prompt"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:layout_marginTop="4dp"
+                android:text="Add nutrients prompt"/>
```

Comment:
Please remove text hardcoding for "Add nutrients prompt".

Reply:
This commit is outdated, please take a look at the latest commit, all string resources have been extracted